### PR TITLE
ant tasks similar to maven plugin

### PR DESCRIPTION
**Pull request by jugu**
Changes made to perform ant tasks similar to maven plugin

### CODE REVIEW

**Comment by jugu** [comment]
Updated changes as mentioned in the review. Please review again!

**Comment by jugu** [comment]
Updated the repo with changes are discussed. Please review again!

File: gp-ant-task/example/build.xml [hunk 1]
```diff
@@ -0,0 +1,81 @@
+<?xml version="1.0" encoding="utf-8"?>
+<project name="gp-ant-task-1.1.5-SNAPSHOT" default="download-translations"
```
Comment (yumaoka) [inline gp-ant-task/example/build.xml]: This is an example project. I'd suggest you to change the project name to "Example Project"
default target is not required. We may simple remove it from <project..>

File: gp-ant-task/example/build.xml [hunk 2]
```diff
@@ -0,0 +1,81 @@
+<?xml version="1.0" encoding="utf-8"?>
+<project name="gp-ant-task-1.1.5-SNAPSHOT" default="download-translations" 
+	xmlns:gp="antlib:com.ibm.g11n.pipeline.ant">
+	
+	<taskdef  uri="antlib:com.ibm.g11n.pipeline.ant" resource="com/ibm/g11n/pipeline/ant/antlib.xml">
+		<!-- If the gp-ant-task jar is not added to ANT_HOME/lib folder, 
+			then you may include the classpath for the jar like in the comment below-->
+		<!--<classpath path="lib/gp-ant-task-1.1.5-SNAPSHOT-with-dependencies.jar"/>-->
```
Comment (yumaoka) [inline gp-ant-task/example/build.xml]: I think it's not a good idea to put a specific version, especially with SNAPSHOT.
I propose to change path value to use gp-ant-task-x.x.x.jar. A user reading the comment should be able to understand x.x.x is a version.

File: gp-ant-task/example/build.xml [hunk 3]
```diff
@@ -0,0 +1,81 @@
...
+	</taskdef>
+	
+	<!--Specify the location of the credentials json file-->
+	<property name="gp.credentials" value="src/credentials.json"/>
+	
+	<!-- basic usecase for uploading resources(this uploads java .properties files by default)-->
+    <target name="upload-resources">
+        <gp:upload credentialsJson= "${gp.credentials}" sourceDir="src/main/resources">
+        </gp:upload>
```
Comment (yumaoka) [inline gp-ant-task/example/build.xml]: Can we specify this in a single line as below?
<gp:upload credentialsJson= "${gp.credentials}" sourceDir="src/main/resources" />

File: gp-ant-task/example/build.xml [hunk 4]
```diff
@@ -0,0 +1,81 @@
...
+    <target name="download-translations">
+        <gp:download credentialsJson="${gp.credentials}" sourceDir="src/main/resources" outputDir="target/classes">
+        </gp:download>
+    </target>
+        	
+    <!-- advanced use case for downloading resources. This downloads java properties files and json properties files and organizes them differently -->
+    <target name="download-translations-adv">
+    	<property name="url" value="https://gp-rest.ng.bluemix.net/translate/rest"/>
+    	<property name="userId" value="b9818b62e1db014edbd2bbb8fae176b4"/>
+  	    <property name="password" value="Tl7edWJzDdV2DRVEBTgnFTAipQdJZ6i4"/>
```
Comment (yumaoka) [inline gp-ant-task/example/build.xml]: Indent? Align the line to other <property> elements

File: gp-ant-task/example/build.xml [hunk 5]
```diff
@@ -0,0 +1,81 @@
...
+    
+    <!-- basic use case for downloading resources (this downloads the java .properties files to the target/classes folder of the project) -->
+    <target name="download-translations">
+        <gp:download credentialsJson="${gp.credentials}" sourceDir="src/main/resources" outputDir="target/classes">
+        </gp:download>
+    </target>
+        	
+    <!-- advanced use case for downloading resources. This downloads java properties files and json properties files and organizes them differently -->
+    <target name="download-translations-adv">
+    	<property name="url" value="https://gp-rest.ng.bluemix.net/translate/rest"/>
```
Comment (yumaoka) [inline gp-ant-task/example/build.xml]: I'd recommend to use prefix "gp" for all of these properties.
gp.url, gp.userId, gp.password, gp.instanceId...

File: gp-ant-task/example/build.xml [hunk 6]
```diff
@@ -0,0 +1,81 @@
...
+        	            bundleLayout="${bundleLayout}"
+        	            outputDir="${download.dest}">
+				<targetLanguage lang="ja"/>
+				<targetLanguage lang="fr"/>
+				<targetLanguage lang="ko"/>
+				<targetLanguage lang="pt-BR"/>
+				<languageMap from="pt-BR" to="pt"/>
+        		<fileset dir="${download.src}" includes="${includepattern}" excludes="${excludepattern}"/>
+       		</bundleSet>
+    		<bundleSet  type="JAVA"
```
Comment (yumaoka) [inline gp-ant-task/example/build.xml]: Indent?

**Pull request merged**